packages/clerk-js/src/utils/querystring.ts

```diff
@@ -1,11 +1,14 @@
-import qs from 'qs';
-
 export const getQueryParams = (queryString: string) => {
-  return qs.parse(queryString || '', {
-    ignoreQueryPrefix: true,
-  }) as Record<string, string>;
-};
+  const queryParamsObject: { [key: string]: string } = {};
+  const queryParams = new URLSearchParams(queryString);
+  queryParams?.forEach((value, key) => {
```

Comment:
`queryParams` is always defined here, no need for `?.`
```suggestion
  queryParams.forEach((value, key) => {
```